src/test/java/com/ebivariation/contigalias/controller/ContigAliasControllerTest.java

```diff
@@ -55,6 +55,6 @@ public void getAssemblyByAccessionGCFNoChromosomes() throws IOException {
         Optional<AssemblyEntity> accession = api.getAssemblyByAccession(GCF_ACCESSION_NO_CHROMOSOMES);
         assertTrue(accession.isPresent());
         List<ChromosomeEntity> chromosomes = accession.get().getChromosomes();
-        assertNull(chromosomes);
+        assertTrue(chromosomes == null || chromosomes.size() == 0);
```

Comment:
I get chromosomes == null when running tests and debugging on my computer as well as on the HTTP request but for some reason when this test is run on GitHub actions, chromosomes is an empty List instead on being null. So I've modified the test and now it passes on GitHub as well.

Reply:
Fixed. Fetching from server returned null and querying db returned null list. Now you get an empty list both ways.